concourse/steps/update_component_deps.py

```diff
@@ -398,8 +399,16 @@ def create_upgrade_pr(
 
     if merge_policy is MergePolicy.MANUAL:
         return
-    # auto-merge - todo: make configurable (e.g. merge method)
-    pull_request.merge()
```

Comment:
what's the default behaviour? if not setting a value will default to repository-default, then we should also add this as fourth value (e.g. `repository-default`)